### DISCUSSION TITLE
Appropriate values for `created_at` and `upated_at` for a new app?

### DISCUSSION THREAD

**Discussion by cori** [runtipi/runtipi-appstore#4427] [Q&A] [answered]
Love to see the audit fields merged and looking forward to the new app view!

I'm in the midst of working on an app config with a drsaft PR on the main repo, and it's now failing CI because I don't have those fields. From the modicum of tracking the feature discussions I've been doing I gather these will be updated by the store / CI code. I can grab the "dummy" values that are in #4425, and i assume those will work, but is that what new app authors _should_ do?

I'm happy to take a stab at updating the docs repo with the new info once i understand it better - i have a local branch with some typo fixes that seemed too trivial to PR on its own that I could include them in (although it'll be a few days most likely).

**Comment by JigSawFr** [comment]
@meienberger nico if you can explain please :) Thanks

**Comment by nicotsx** [comment, marked as answer]
Hello, thanks for asking. You can check out the reference [config.json guide](https://runtipi.io/docs/reference/config-json) which explains how to generate these values. I bet we could improve the CI to automatically add it tho

**Comment by cori** [comment]
hah that docs update snuck in after i posted the question and matches my expectations.
